README.md

```diff
@@ -52,10 +52,181 @@ The setup is done in 3 primary steps.  You will download the code, setup the app
 1. [Clone the repo](#1-clone-the-repo)
 2. [Create Watson Studio Project](#2-create-watson-studio-project)
 3. [Add and Refine data asset into Watson Studio](#3-add-and-refine-data-asset-into-watson-studio)
-3. [Create DB2 warehouse on cloud and add the connection to Watson Studio](#3-create-db2-warehouse-on-cloud-and-add-the-connection-to-watson-studio)
-4. [Create Apache Spark as a service with IBM Cloud](#4-create-apache-spark-as-a-service-with-ibm-cloud)
-5. [Create Watson Machine Learning with IBM Cloud](#4-create-watson-machine-learning-with-ibm-cloud)
-6. [Add new Watson Machine Learning Model to Watson Studio](#6-add-new-watson-machine-learning-model-to-watson-studio)
-7. [Add Feedback data and new evaluations to the continuiously learning model](#7-add-feedbasck-data-and-new-evaluatioins-to-the-continuiously-learning-model)
-8. [Deploy the model to expose an API](#8-deploy-the-model-to-expose-an-api)
-9. [Test the model](#9-test-the-model)
+4. [Create DB2 warehouse on cloud and add the connection to Watson Studio](#3-create-db2-warehouse-on-cloud-and-add-the-connection-to-watson-studio)
+5. [Create Apache Spark as a service with IBM Cloud](#4-create-apache-spark-as-a-service-with-ibm-cloud)
+6. [Create Watson Machine Learning with IBM Cloud](#4-create-watson-machine-learning-with-ibm-cloud)
+7. [Add new Watson Machine Learning Model to Watson Studio](#6-add-new-watson-machine-learning-model-to-watson-studio)
+8. [Add Feedback data and new evaluations to the continuiously learning model](#7-add-feedbasck-data-and-new-evaluatioins-to-the-continuiously-learning-model)
+9. [Deploy the model to expose it through an API](#8-deploy-the-model-to-expose-it-through-an-api)
+10. [Test the model](#9-test-the-model)
+
+
+###1. Clone the rep
+
+Clone the `continuous-learning-with-watson-ml-and-db2` locally. In a terminal, run:
+
+```
+$ git clone https://github.com/IBM/continuous-learning-with-watson-ml-and-db2
+$ cd continuous-learning-with-watson-ml-and-db2
+```
+
+###2. Create Watson Studio Project
+
+If you do not already have a IBM Cloud account, [sign up for Bluemix](https://console.bluemix.net/registration).
+
+Login to IBM Cloud account, from the catalog search for `Watson Studio`, select it, choose the `lite` plan and click `create`.
+
+![](doc/source/images/watson-studio-1.png)
+
+![](doc/source/images/watson-studio-2.png)
+
+Create a new Project by clicking the `New Project` link, choose `Complete` , give it a name and click create.
+
+> you might have to create `Cloud Object Storage` if you don't already have one.
```

Comment:
I'd explicitly add this to the directions before creating the watson studio project